blockchain/chain_test.go

```diff
@@ -1311,3 +1313,381 @@ func TestIsAncestor(t *testing.T) {
 			branch2Nodes[0].hash.String())
 	}
 }
+
+func TestInvalidateBlock(t *testing.T) {
+	tests := []struct {
+		name     string
+		chainGen func() (*BlockChain, []*chainhash.Hash, func())
+	}{
+		{
+			name: "One branch, invalidate once",
+			chainGen: func() (*BlockChain, []*chainhash.Hash, func()) {
+				source := rand.NewSource(time.Now().UnixNano())
+				rand := rand.New(source)
+
+				chain, params, tearDown := utxoCacheTestChain("TestInvalidateBlock-one-branch-invalidate-once")
```

Comment:
I know we don't have the full linter active in this project, but can we wrap all this to 80 char columns to match most of the rest of the codebase?

Reply:
Addressed in the latest pushes.